css/popup-window.css

```diff
@@ -0,0 +1,224 @@
+.modal {
+  position: fixed;
+  z-index: 1;
+  background-color: rgb(255, 255, 255, 0.9);
+  backdrop-filter: blur(9px);
+  overflow-y: hidden;
+  overflow-x: hidden;
+  width: 80%;
+  height: auto;
+  margin: 114px auto;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 24px;
+  left: 10%;
+  top: -5%;
+}
```

Comment:
- [ ] At the moment of opening the app the modal window appears, please 🙏🏽  fix this issue for a better UI. 
- [ ] The modal is contained in another div with opacity, a light grey container, as in the desktop version as in the mobile version, please add it to be similar to the Figma template 👇🏽 
- [ ] Please 🙏🏽 ensure that all the modal is centered and the content is able to see by the user, we could use `overflow-y: scroll;` for a better UI.

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/232169235-f007111e-71c0-4965-a304-3bc6c8644b12.png) | ![image](https://user-images.githubusercontent.com/58642949/232169272-749e0142-efe1-400d-bc1d-66b2a7a88984.png) |
| ![image](https://user-images.githubusercontent.com/58642949/232171044-c104fb37-8335-41bf-aa77-f56f8d03c1d2.png) | ![image](https://user-images.githubusercontent.com/58642949/232171013-ae6430eb-9d6d-4663-8205-6459c60c4a50.png) |